src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -21,7 +20,7 @@
 from qibolab.sweeper import Parameter, Sweeper, SweeperType
 
 
-class ClusterQCM_BB(Instrument):
+class ClusterQCM_BB(ClusterModule):
```

Comment:
Side note, for a further PR:
- the name `Cluster` in `ClusterQCM_BB` is redundant, because it inherits from `ClusterModule`, and `QCM_BB` is already enough to identify it
- `QCM_BB` does not respect the conventions on class naming, because of the underscores, the valid alternatives are `QCMBB`, or (the more recommended) `QcmBb` (it is quite customary to treat acronyms as regular words in identifiers)

Reply:
Opened issue https://github.com/qiboteam/qibolab/issues/742